numpy/core/src/npymath/npy_math.c.src

```diff
@@ -130,7 +130,7 @@ double npy_atan2(double y, double x)
         return npy_atan(y);
     }
 
-    m = 2 * npy_signbit(x) + npy_signbit(y);
+    m = 2 * (npy_signbit((x)) != 0 ? 1 : 0) + (npy_signbit((y)) != 0 ? 1 : 0);
```

Comment:
Actually, IIRC comparison operators are documented to return 1 or 0, so this can be simplified to
```
m = 2 * (npy_signbit((x)) != 0) + (npy_signbit((y)) != 0);
```
>+----C99:6.5.8 Relational operators-----
>6. Each of the operators < (less than), > (greater than), <= (less than or equal to), and >= (greater than or equal to)
   shall yield 1 if the specified relation is true and 0 ifit is false.The result has type int.

That goes back to K&R.